src/components/checkbox/index.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import { Icon } from '../index';
+
+const Checkbox = (props) => (
+  <label
+    className={ cx('checkbox-wrapper', {
+      'checkbox-wrapper--checked': props.checked
+    }) }
+    onClick={ (e) => {
+      e.preventDefault();
+      props.onChange();
+    } }
+  >
+    <div className="checkbox">
+      { props.checked && (
```

Comment:
It's fine but wouldn't it be better to set the visibility of the checkmark via css? That would be the way if we wanna implement some transitions on it and it performs better (I guess, can't give you numbers :))

Reply:
True, I'll fix that!